pandevice/base.py

```diff
@@ -477,13 +478,20 @@ def element(self, with_children=True, comparable=False):
 
         return root
 
-    def element_str(self):
+    def element_str(self, pretty_print=False):
         """The XML representation of this PanObject and all its children.
 
+        Args:
+            pretty_print (bool): Return the resulting string pretty_printed with indentation.
+
         Returns:
             str: XML form of this object and its children
 
         """
+        if pretty_print:
+            raw = ET.tostring(self.element, encoding="utf-8")
```

Comment:
Should probably be `self.element()`..?

I think this also means that you need to add at least one test that uses the new `pretty_print` arg.